### ISSUE TITLE
How long will the data generation take?

### ISSUE THREAD

**Issue by youdutaidi** [zkkli/PSAQ-ViT#1]
Thank you for  sharing your brilliant work ~Could you share the information about your  total training  time of every stage, especially about the data generating time

**Comment by WhereKey** [comment]
Yeah, I have the same question about PSAQ-ViT v2. There is no auxiliary category prior for data generation.  I think the generator needs much more iterations to learn the differences of all categories.

**Comment by zkkli** [comment]
Hi,

The whole process takes less than 4 min on an RTX 3090 GPU and most time is spent in the image generation stage, since the parameter calibration without training produces small overhead.

For instance, the specific timing of each stage on DeiT-B model  is as follows:
  - Image Generation: 227s
  - Quantization Calibration: 0.17s

**Comment by WhereKey** [comment]
I have to say your work is very nice. 
Most data-free quantization codes(even published  on CVPR) lost quantization and do not fully use low-precision input and low-precision weight for matrix multiplication.
But your work is to do it completely.👍🏻
I've learned a lot from your codes!
